### PR TITLE
Create index.d.ts

### PR DESCRIPTION
**Pull request by YahuiWong**

### CODE REVIEW

**Comment by kirankalyan5** [comment]
@YahuiWong Could add a brief description of this PR.

P.S I do not have much knowledge on ts.

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        // Default  [0];Array of indices of tab items to be selected initially - when multiple prop is true else it will take selectedIndex
+        selectedIndices?: Array<number>
+        // Default  true; Boolean to enable or disable the component
+        enable?: boolean
+        // Default  false; Boolean which enables the multiple selection option
+        multiple?: boolean
+        // Default  5;borderRadius of whole tab
+        borderRadius?: number
+        // Default  base styles added in SegmentedControlTab.js;external styles can be passed to override the default styles of the segmentedControl wrapper
+        tabsContainerStyle?: any
```
Comment (kirankalyan5) [inline index.d.ts]: `StyleProp<ViewStyle>` instead of any?

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        // Default  default style opacity: 0.6;Custom style that can be passed when enable is set to false
+        tabsContainerDisableStyle?: any
+        // Default  base styles added in SegmentedControlTab.js;external styles can be passed to override the default styles of the tabs
+        tabStyle?: any
+        // Default  base styles added in SegmentedControlTab.js;external styles can be passed to override the default styles of the tab title
+        tabTextStyle?: any
```
Comment (kirankalyan5) [inline index.d.ts]: `StyleProp<TextStyle>` instead ?

File: index.d.ts (hunk 1 above)
Comment (YahuiWong) [reply index.d.ts]: This is a good idea, I am starting to make a ts d.ts, because this file I can already use, I hope to be better

**Comment by kirankalyan5** [comment]
Thanks for the PR @YahuiWong  💪

**Pull request merged**